### DISCUSSION TITLE
Recording live streams from youtube

### DISCUSSION THREAD

**Discussion by ghost** [streamlink/streamlink#4422] [Q&A] [answered]
It seems to be working fine on my pc so unsure whats different as they're the same versions and the same command but on my laptop i am getting this
![ueJSj6M](https://user-images.githubusercontent.com/102879623/161400398-5adeb66b-2260-4d16-8be2-f8f19738f25d.png)

Does anyone know what might be causing it?

**Comment by mkbloke** [comment, marked as answer]
https://github.com/streamlink/streamlink/issues/4211#issuecomment-980283012
